src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write code here
+const element = document.querySelector('.spider');
+
+const parentElement = document.querySelector('.wall');
```

Comment:
Consider renaming the variable 'parentElement' to 'wallElement' for better readability.